src/components/panel/panel.spec.js

```diff
@@ -650,6 +650,83 @@ describe('$mdPanel', function() {
     });
   });
 
+  describe('animation logic', function() {
+    var config;
+    var mdPanelAnimation;
+    var myButton;
+
+    beforeEach(function() {
+      config = {
```

Comment:
Your other PR was merged, so you use the DEFAULT_CONFIG instead. (You may just need to rebase to get that change into this client.)